snap/snapcraft.yaml

```diff
@@ -8,7 +8,7 @@ description: |
 
 grade: stable
 confinement: classic
-base: core18
+base: core20
```

Comment:
I don't think it can go to core20 yet. I'll check out what is going on with snap. Do you know if the curl version you prescribed is actually required, or can it go older?

Reply:
I'll admit I don't know the nuances of the snap base snap -- it's all a bit confusing.

Reply:
https://snapcraft.io/docs/base-snaps explains it a bit